snyk/cli
# docs: update cli readme workflow

#### What does this PR do?

This PR introduces a GitHub Actions workflow to automatically synchronize content from the Snyk CLI's GitBook documentation to the CLI repository's README. Additionally, the `README.md` has been manually updated to match the content of the GitBook documentation.

#### Where should the reviewer start?

The primary addition is the GitHub Actions workflow file in `.github/workflows/`. This file contains the script that handles the synchronization process. Reviewers should also note the changes made directly to `README.md`.

#### How should this be manually tested?

1. Trigger the GitHub Action manually or wait for the scheduled run time.
2. Check the `README.md` in the destination branch (`docs/automatic-gitbook-update`) to see if it has been updated with content from the GitBook documentation.

#### Any background context you want to provide?

The goal of this synchronization is to ensure that the `README` in the CLI repository is always up-to-date with the latest documentation available on GitBook, reducing redundancy and ensuring accuracy.

#### What are the relevant tickets?

N/A

#### Screenshots

N/A


#### Additional questions

N/A
